server/src/test-fast/java/com/thoughtworks/go/config/ConfigConverterTest.java

```diff
@@ -1631,7 +1631,7 @@ public void shouldConvertFetchArtifactTaskWhenSourceIsDirectoryToCR() {
                new CaseInsensitiveString("upstream"),
                new CaseInsensitiveString("stage"),
                new CaseInsensitiveString("job"),
-               "src",
```

Comment:
@ibnc Shouldn't this be verified by adding a line:

```
fetchTask.setSrcdir("src")
```

Otherwise, the test name doesn't reflect what's happening.

Reply:
nvm. Changed the wrong test :/